src/ansys/grantami/recordlists/_connection.py

```diff
@@ -1063,10 +1086,10 @@ def search_for_audit_log_entries(
 class _ItemResolver:
     _max_requests = 5
 
-    def __init__(self, client: ApiClient, read_mode: bool) -> None:
-        self._record_histories_api = v2025r2api.RecordsRecordHistoriesApi(client)
-        self._record_versions_api = v2025r2api.RecordsRecordVersionsApi(client)
-        self._db_schema_api = v2025r2api.SchemaDatabasesApi(client)
+    def __init__(self, client: ApiClient, read_mode: bool, api_module: types.ModuleType) -> None:
+        self._record_histories_api = api_module.RecordsRecordHistoriesApi(client)
```

Comment:
We should really have done this previously, it's just that the responses for the API calls we used didn't change. Now one of the models has changed, we need to make sure the api versions we use are in sync with the models bound to the client.